modules/subscriptions.php

```diff
@@ -193,7 +193,7 @@ function update_published_message( $messages ) {
 
 		$view_post_link_html = sprintf( ' <a href="%1$s">%2$s</a>',
 			esc_url( get_permalink( $post ) ),
-			__( 'View post' ) // intentinally omitted domain
+			__( 'View post', 'jetpack' )
```

Comment:
Asked about this here: 
https://github.com/Automattic/jetpack/pull/9081/files#r199159936

Reply:
Chatted with Enej about this:

Enej
> Yes core already translates that string. In a very similar context.

Mikael
>I think it’ll get pulled up by GlotPress as a translation suggestion, even if we have ‘jetpack’ text-domain there
>i.e. just for one string, consistency might be better than the trouble of translating it

Enej
> You make a good point. I do think it would be good to have a mechanism where our script knows not to translate something.
>
>We could create an issue for it as something to change in the future. And cc some of our translation folks. But let's add the Jetpack text domain for this one string.